### PR TITLE
Added a button to remove a tag.

### PR DESCRIPTION
**Pull request by CasuaIAz**
Type tag name and click the remove button to delete the tag.

### CODE REVIEW

**Comment by t-eliks** [comment]
Having to manually type the tag in order to delete it is terrible UX. Why not atleast make it so when you click on a tag, it gets selected and then you can press the remove button to remove it?

**Comment by Kodnot** [comment]
Merge with newest dev branch and fix interactions with filtering:
- When a tag is removed, tag list for Select tags is not updated. It's only updated when a new tag is added, or the program is reloaded.
- If the user filters by tag and then removes that tag, the list of notes is not updated (eg. note A has a tag "dik", user filters by "dik", removes "dik" tag from note A, note is still shown. The list is is only refreshed after a tag is checked/unchecked).

File: ToNote/Views/NoteView.xaml [hunk 1]
```diff
@@ -51,10 +51,10 @@
                     <ItemsControl ItemsSource="{Binding Tags}" Grid.Row="1">
                         <ItemsControl.ItemTemplate>
                             <DataTemplate DataType="{x:Type System:String}">
-                                <Border Style="{StaticResource BorderStyle}">
+                                <Button Click="Button1_Click">
                                     <TextBlock Text="{Binding}" Style="{StaticResource TagTextBlockStyle}">
```
Comment (Kodnot) [inline ToNote/Views/NoteView.xaml]: `<TextBlock Text="{Binding}" Style="{StaticResource TagTextBlockStyle}" />`

File: ToNote/Views/NoteView.xaml.cs [hunk 2]
```diff
@@ -43,5 +44,16 @@ public NoteView()
                 e.Handled = true;
             };
         }
+        private void Button1_Click(object sender, System.Windows.RoutedEventArgs e)
+        {
+            Note note = (Note)DataContext;
+
+            var tagName = ((Button)sender).DataContext.ToString();
```
Comment (Kodnot) [inline ToNote/Views/NoteView.xaml.cs:51]: All of this breaks MVC. This should be done with commands, like adding tags and everything else. Consult @t-eliks on how to if you get stuck.

**Review by Kodnot — CHANGES_REQUESTED** [review]

**Comment by Kodnot** [comment]
[Allow user to remove tags](https://trello.com/c/KhwxGEDW/71-allow-user-to-remove-tags)

**Pull request closed without merging**